index.html

```diff
@@ -17,5 +17,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
```

Comment:
Remove development comments like this as they will be rendered in the browser and will be visible in the source code.